### ISSUE TITLE
Can't open database

### ISSUE THREAD

**Issue by ghost** [PrabhdeepSingh/WurmUnlimitedAdmin#7]
SQLSTATE[HY000] [14] unable to open database file

**Comment by ghost** [comment]
alright i get past that part, however, now i get sent to http://localhost/server/select/ and nothing happens (blank page)

**Comment by ghost** [comment]
2017/02/17 11:18:19 [error] 504606#504606: *119 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined index: serverList in /var/www/localhost/htdocs/website/server/select/index.php on line 44" while reading response header from upstream, client: 192.168.0.1, server: maynard.mystrix.3d-game.com, request: "GET /server/select/ HTTP/1.1", upstream: "fastcgi://unix:/var/www/localhost/htdocs/php-fpm.sock:", host: "192.168.0.1"

**Comment by PrabhdeepSingh** [comment]
Did you edit the config.php file in the includes folder to add your server?

**Comment by ghost** [comment]
Yes, i've tried all sorts of things... being a php scripter myself i've looked over your code.. there is no file called wua.js anywhere in the project
you have variables that aren't being properly set in your session... incomplete documentation... missing variables where you claim they are. and you can bypass logging in by hitting refresh on the log in page because of improper session management.
This project is a mess
